app/src/main/java/org/gdg/frisbee/android/arrow/ArrowActivity.java

```diff
@@ -160,11 +168,53 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     protected void onResume() {
         super.onResume();
-        handleIntent(getIntent());
 
         if (!PrefUtils.isSignedIn(this)) {
             finish();
         }
+
+        handleIntent(getIntent());
+    }
+
+    private void checkSnapshotUpgrade() {
+        Games.Snapshots.open(getGoogleApiClient(), Const.GAMES_SNAPSHOT_ID, false).setResultCallback(
+                new ResultCallback<Snapshots.OpenSnapshotResult>() {
+                    @Override
+                    public void onResult(Snapshots.OpenSnapshotResult openSnapshotResult) {
```

Comment:
Class also implements this interface I think. And it is not used. It is empty at the bottom. Maybe we can move this one to there.